src/mcts/search.cc

```diff
@@ -1248,13 +1285,18 @@ void SearchWorker::DoBackupUpdateSingleNode(
     if (can_convert) {
       p->MakeTerminal(v == 1.0f ? GameResult::BLACK_WON
                                 : v == -1.0f ? GameResult::WHITE_WON
-                                             : GameResult::DRAW);
+                                             : GameResult::DRAW, false);
     }
 
     // Q will be flipped for opponent.
     v = -v;
     depth++;
 
+    if (depth_parity) {
+        m = m + 1.00f;
+    }
+    depth_parity = !depth_parity;
+
```

Comment:
Can you explain what `depth_parity` is supposed to be?

Reply:
Move number increases only on white's move, `depth_parity` keeps track of who's turn it is.

Reply:
Moves left head outputs moves not plies. I did it that way because moves are stored in the training data, not plies. Changing it to plies would require counting the plies in rescorer. There is space for one byte so it would change the maximum moves left prediction from 256 moves to 256 plies.

Reply:
New version is in plies.